rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -465,9 +465,13 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   rclcpp::Clock::SharedPtr
   get_clock();
 
+  RCLCPP_PUBLIC
```

Comment:
This macro name is wrong. It needs to be `RCLCPP_LIFECYCLE_PUBLIC`

See warning on Windows: https://ci.ros2.org/job/ci_windows/8722/warnings43Result/new/